src/main/java/de/zalando/aruha/nakadi/domain/EventTypeStatistics.java

```diff
@@ -0,0 +1,73 @@
+package de.zalando.aruha.nakadi.domain;
+
+import java.util.Objects;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+public class EventTypeStatistics {
+    @NotNull
+    @Min(value = 0, message = "expected_write_rate can't be less then zero")
+    private Integer expectedWriteRate; // messages per minute
```

Comment:
@v-stepanov I think this name comes from the API. If we conclude that API names are not clear, then I would suggest to change it as well.